contracts/token/ERC20/ERC20Detailed.sol

```diff
@@ -10,13 +10,34 @@ import "./IERC20.sol";
  * just as on Ethereum all the operations are done in wei.
  */
 contract ERC20Detailed is IERC20 {
-  string public name;
-  string public symbol;
-  uint8 public decimals;
+  string private name_;
+  string private symbol_;
+  uint8 private decimals_;
```

Comment:
Is this a deviation from the standard?

Reply:
Yes! We need the public getters.

Reply:
Ok, I'll rename the getters I added from `getName` to `name`, and such.